### DISCUSSION TITLE
IPTV on the UDM/P - will it also work if you don't have the pro model?

### DISCUSSION THREAD

**Discussion by Boetros** [fabianishere/udm-iptv#11] [Q&A] [answered]
@fabianishere 

I have a question on your article on "IPTV on the UDM/P", which is a great article btw.

Will the instructions also work if the UDM is not a pro model (I have a regular UDM = https://dl.ubnt.com/ds/udm_ds)? Is this what you refer to when you mention "If you have the UDM base model, please update the WAN port from eth8 to eth4."?

FYI: I do have a Unify switch linked to it.

Thank you for your reply.

**Comment by fabianishere** [comment, marked as answer]
Yes, any mention of "UDM base model" in the document refers to the regular UDM you linked to.
